gax/src/test/java/com/google/api/gax/rpc/RetryingTest.java

```diff
@@ -29,6 +29,8 @@
  */
 package com.google.api.gax.rpc;
 
+import static com.google.common.truth.Truth.assertThat;
```

Comment:
Don't static import this since there are at least three different assertThat's this could be